knowledge-base/daterangepicker-css-customization.md

```diff
@@ -0,0 +1,97 @@
+---
+title: How to Customize the DateRangePicker
+description: Learn how to enhance the Blazor DateRangePicker by customizing its styling, disabling weekends, reducing the width of DateInputs, preventing horizontal expansion, hiding Start and End labels, and making the component more compact for better user experience and design consistency in your application.
+type: how-to
+page_title: How to Customize the DateRangePicker
+slug: daterangepicker-kb-css-customization
+tags: daterangepicker, styling
+ticketid: 1633582
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>DateRangePicker for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+This KB article answers the following questions:
+
+* How to customize the Blazor DateRangePicker styling?
+* How to make weekends disabled in the DateRangePicker component?
+* How to reduce the width of the two DateInputs?
+* How to prevent horizontal expansion of the DateRangePicker?
+* How to hide the Start and End labels of the DateRangePicker?
+* How to make the DateRangePicker more compact?
+
+## Solution
+
+1. Use a `Class` parameter and custom CSS to override our theme and hide the labels (**.k-floating-label**). Note that If it is used **display:none**, it violates the accessibility compliance. Instead, use **font-size:0** for example.
+2. Override the default width of **10em** of the two `DateInputs` (**.k-dateinput**).
+3. Change the display of the element to inline with **display:inline-flex** to prevent the horizontal expansion of the component.
```

Comment:
```suggestion
3. Change the display of the element to inline by using `display:inline-flex` to prevent the horizontal expansion of the component.
```